query-connector/src/app/query/components/CustomizeQuery.tsx

```diff
@@ -72,13 +72,15 @@ const CustomizeQuery: React.FC<CustomizeQueryProps> = ({
   }, [queryValueSets]);
 
   // Compute counts of each tab-type
-  const countLabs = countDibbsConceptTypeToVsMapItems(valueSetOptions.labs);
-  const countConditions = countDibbsConceptTypeToVsMapItems(
-    valueSetOptions.conditions,
-  );
-  const countMedications = countDibbsConceptTypeToVsMapItems(
-    valueSetOptions.medications,
-  );
+  const countLabs = valueSetOptions.labs
```

Comment:
there was a null pointer error here that occurred when there were empty fetched concept types, so added these checks to have the page render correctly.